docs/code/getting-started.md

```diff
@@ -523,7 +523,7 @@ Now it's time to create the rules that will allow your app to be built as a .deb
 
           * Initial Release.
 
-         -- Your Name <you@emailaddress.com>  Friday, 20 Apr 2018 04:53:39 -0500
+         -- Your Name <you@emailaddress.com>  Friday, 20 Apr 2019 04:53:39 -0500
```

Comment:
This date is in the future...

Reply:
I suppose this date is kind of impossible to get right seeing as its more specific than just the year, but I guess changing it to today for now wont hurt.

```suggestion
         -- Your Name <you@emailaddress.com>  Monday, 8 Apr 2019 04:53:39 -0500
```

Reply:
@bunnyapocalypse, you will need to merge that suggestion. I don't have permission to edit your branch